docs/apis-tools/zeebe-api/deprecated-rpcs.md

```diff
@@ -0,0 +1,87 @@
+---
+id: deprecated-rpcs
+title: "Deprecated RPCs"
+slug: /apis-tools/zeebe-api/gateway-service
```

Comment:
```suggestion
slug: /apis-tools/zeebe-api/deprecated-rpcs
```

I'm not sure if I guessed correctly what it _should_ be, but the current value is a duplicate (of the docs/apis-tools/zeebe-api/gateway-service.md doc). I think that's what's causing your current build error of `Warning:  Duplicate routes found!`

Reply:
Perfect! Thank you for checking for me. Needed your second set of eyes.